Library/Formula/i2p.rb

```diff
@@ -0,0 +1,24 @@
+class I2p < Formula
+  desc "I2P is an anonymous overlay network - a network within a network"
+  homepage "https://geti2p.net"
+  url "https://download.i2p2.de/releases/0.9.21/i2pinstall_0.9.21.jar"
+  sha256 "0238ffc6ea44099ef4fe6c20913cb4eec675c2760aea07dfe7d499addcc89cf2"
+
+  depends_on :java => "1.6+"
+
+  def install
+    (buildpath/"path.conf").write "INSTALL_PATH=#{libexec}"
+    system "java", "-jar", "i2pinstall_#{version}.jar", "-options", "path.conf"
+    wrapper_name = "i2psvc-macosx-universal-#{MacOS.prefer_64_bit? ? '64' : '32'}"
+    libexec.install_symlink libexec/wrapper_name => "i2psvc"
```

Comment:
It's a nit, but if we could leave one blank space between the compile stage and the installation stage that'd be nice. Having chunked together `def install` blocks makes them a little unpleasant to read.